cal_ratio_trainer/common/trained_model.py

```diff
@@ -95,3 +96,15 @@ def load_model(info: str, base_path: Path = Path(".")) -> Model:
     name, run = info.split("/")
     model_dir = find_training_result(name, int(run), base_path)
     return _load_model_only(model_dir)
+
+
+def load_model_from_spec(spec: epoch_spec) -> TrainedModel:
+    """Given a model spec, load the file and return it.
```

Comment:
Ok - nice - search the source base for other occurences of both of these methods used together like this, and replace them with a call to this. Otherwise we'll have both this nice method and those two lines of code. :-)